SPECS/mstflint/mstflint.spec

```
@@ -40,13 +52,22 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+# Compile mstflint driver
+cd kernel
+EXTRA_CFLAGS='-DVERSION=\"%{version}\"' make KPVER=%{kver}
+
 %install
 %make_install
 # Remove the devel files that we don't ship
 rm -fr %{buildroot}%{_includedir}
 find %{buildroot} -type f -name "*.la" -delete -print
 find %{buildroot} -type f -name '*.a' -delete
 
+export INSTALL_MOD_PATH=%{buildroot}
+cd kernel
+install -d $INSTALL_MOD_PATH/%{install_mod_dir}
+cp $PWD/mstflint_access.ko $INSTALL_MOD_PATH/%{install_mod_dir}
```

Similar clean-up as in the "signed" spec:
```suggestion
install -dm 755 %{buildroot}%{install_mod_dir}
install -m 644 kernel/mstflint_access.ko %{buildroot}%{install_mod_dir}
```

---

Could we do still do this clean-up? I see we already have that in the "signed" spec.

---

Done!